crypto/cmp/cmp_asn.c

```diff
@@ -332,6 +351,243 @@ int OSSL_CMP_ITAV_get0_rootCaKeyUpdate(const OSSL_CMP_ITAV *itav,
     return 1;
 }
 
+OSSL_CMP_ITAV
+*OSSL_CMP_ITAV_new0_crlStatusList(STACK_OF(OSSL_CMP_CRLSTATUS) *crlStatusList)
+{
+    OSSL_CMP_ITAV *itav;
```

Comment:
```suggestion
    OSSL_CMP_ITAV *itav = NULL;
```

Reply:
Unnecessary as itav is assigned below immediately.